src/platform/user/profile/vap-svc/components/base/VAPServiceEditModalErrorMessage.jsx

```diff
@@ -16,7 +17,6 @@ function hasError(codes, errors) {
 
 export default function VAPServiceEditModalErrorMessage({
   error: { errors = [] },
-  clearErrors,
```

Comment:
If you're going to totally remove support for this prop can you remove all instances where this prop is used?

Reply:
I'd suggest leaving support for it, just don't provide the prop in your case where you don't want it.